bugwarrior/docs/services/gmail.rst

```diff
@@ -33,6 +33,7 @@ Here's an example of a gmail target:
     service = gmail
     gmail.query = label:action OR label:readme
     gmail.login_name = you@example.com
+    gmail.project_name = example_gmail
```

Comment:
I know the other services with this feature document it like this but I think it would be better to add a separate section in the docs about it. It seems like we've moved towards trying to keep the initial "Example Service" section as a minimal usage example or at least just the most common options.

Reply:
Ok sure, will make the changes.